server/src/septic/septicCnfg.ts

```diff
@@ -128,19 +128,18 @@ export function extractXvrRefs(obj: SepticObject): SepticReference[] {
         return [];
     }
 
-    if (objectDef.refs.identifier) {
-        if (obj.identifier) {
-            let ref: SepticReference = {
-                identifier: obj.identifier.name,
-                location: {
-                    uri: "",
-                    start: obj.identifier.start,
-                    end: obj.identifier.end,
-                },
-                obj: obj,
-            };
-            xvrRefs.push(ref);
-        }
+    if (objectDef.refs.identifier && obj.identifier) {
+        let isXvr = /[\w]*[TMECDP]vr$/.test(obj.type);
```

Comment:
I have never seen a Pvr in a cfg file, except for CalcPvrs. I suggest to check with @mortenfre whether to include CalcPvr in this context and whether there are other Pvrs that may be of relevance.